ansible/roles_ocp_workloads/ocp4_workload_smartvillage/tasks/workload.yml

```diff
@@ -84,6 +84,52 @@
     - kustomize/overlays/sandbox/iotagentjsons/
     - kustomize/overlays/sandbox/smartabyarsmartvillages/
 
+- name: Ensure smartvillage namespace topology is visible to users
+  hosts: localhost
+  gather_facts: no
```

Comment:
true and false are preferred. Yes and no are discouraged and will fail the linter